server/sandbox.go

```diff
@@ -599,3 +618,18 @@ func getSELinuxLabels(selinuxOptions *pb.SELinuxOption) (processLabel string, mo
 	}
 	return label.InitLabels(label.DupSecOpt(processLabel))
 }
+
+// sysctlsFromPodAnnotation parse annotation in form of comma separated list of sysctl_name=value key-value pairs, returns a list of {Name, Value} struct.
+func sysctlsFromPodAnnotation(annotation string) ([]sysctl, error) {
```

Comment:
Yep, I am ok with that, just we are not sure whether the logic [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/helpers.go#L611), which imply value of sysctl can be empty, is applicable in cri-o.